Misc/python.man

```diff
@@ -177,6 +180,9 @@ adding .opt-1 before the .pyc extension.
 Do \fB-O\fP and also discard docstrings; change the filename for
 compiled (bytecode) files by adding .opt-2 before the .pyc extension.
 .TP
+.B \-P
+Don't add sys.path[0].
```

Comment:
consistent with elsewhere, reword this to describe the explicit details and mention to see also the PYTHONSAFEPATH environment variable.

"Don't automatically prepend a potentially unsafe path to sys.path such as the current directory, script directory or empty string. See also: The PYTHONSAFEPATH environment variable."  or similar?

Reply:
Oh, I updated PYTHONSAFEPATH in the man page, but I forgot to update -P. Fixed.